docs/modules/ROOT/pages/tmail-backend/jmap-extensions/jmapFilters.adoc

```diff
@@ -479,7 +480,7 @@ Server returns:
     "list": [
       {
         "id": "flt-task-001",
-        "mailboxId": "mailboxId123",
+        "mailboxName": "mailbox123",
```

Comment:
Reason: we can not resolve `mailboxId` from `mailboxName` if the mailbox was deleted.
Frontend can use `mailboxName` to resolve the mailbox and know if the mailbox was deleted if needed.

Otherwise, we can add `mailboxId` to the task's additional information. Thoughts?

Reply:
Ok. I will drop the latest commit.

> Otherwise, we can add mailboxId to the task's additional information

We will follow up that in a later PR.